BUILDING.md

```diff
@@ -119,3 +119,50 @@ See the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setup guide for
 ## Publish your image
 
 See the [PUBLISHING](PUBLISHING.md) guide for information on deploying Bottlerocket images and repositories.
+
+## Building out-of-tree kernel modules
+
+To further extend Bottlerocket, you may want to build extra kernel modules.
+This section will be filled out more in the future, but the first step is to download the "kmod kit" that contains the kernel headers and toolchain you'll need to use.
```

Comment:
Do we need to say "This section will be filled out more in the future"? We're always interested in pull requests that expand docs in useful ways.

Reply:
I wanted to say something, because it's pretty clear this section isn't useful on its own without a fair amount of outside context.